crap-advisor-frontend/src/core/components/RestroomMarker/index.jsx

```diff
@@ -4,15 +4,22 @@ import L from "leaflet";
 import restroomImage from "../../../assets/restroomMarker.png";
 import "./index.css"
 
-const RestroomMarker = ({restroom}) => {
+const RestroomMarker = ({restroom, selectRestroom}) => {
     const icon = L.divIcon({
         className: "icon",
-        html: `<div class="marker-container">` +
-            `<img class="marker-icon" src='${restroomImage}' alt='Toilet'/><p class="marker-text outlined">${restroom.rating}</p></div>`
+        html:
+            `<div class="marker-container">` +
+            `<img class="marker-icon" src='${restroomImage}' alt='Toilet'/>` +
+            `<p class="marker-text outlined">${restroom.rating != null ? restroom.rating : ""}</p>` +
```

Comment:
What about `<p class="marker-text outlined">${restroom.rating && restroom.rating}</p>` ? It's will be working well?

Reply:
I don't want NULL to be visible in restroom marker, changed to restroom.rating ?? ""